@here/harp-vectortile-datasource/lib/VectorTileDecoder.ts

```diff
@@ -317,11 +300,16 @@ export class VectorTileDecoder extends ThemedTileDecoder {
         projection: Projection
     ): Promise<DecodedTile> {
         const startTime = PerformanceTimer.now();
+        const dataAdapter = this.getDataAdapter(data);
```

Comment:
Does this need to be called for every tile key? Could it be lazy initialized? Perhaps not if the class has to support different formats at run time.

Reply:
Changed to do lazy initialization.